modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,30 @@
+Housekeeping and workloads may utilize the central processing units (CPUs) in a way that would impact latency sensitive processes running on those CPUs. Partitioning your workloads on separate CPUs prevents the processes from interfering with each other. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
 
-You can ensure that housekeeping tasks and workloads run on separate NUMA nodes by specifying two groups of CPUs in the `spec` section of the performance profile. 
+The exact partitioning pattern to use depends on many factors like hardware, workload characteristics and system load. Some sample use cases are as follows:
+
+* If the latency sensitive workload uses specific hardware (like network interface cards (NICs)), ensure that the CPUs in the isolated pool are as "close" to this hardware as possible. Place the workload in at least the same Non-Uniform Memory Access (NUMA) node.
+
+* The reserved pool is used for handling all interrupts, but not all interrupt requests generate noise. When depending on system networking, allocate a sufficiently sized reserve pool to handle all the incoming packet interrupts.
+
+The above examples require detailed analysis and measurements to be combined into a pattern where all housekeeping CPUs are allocated from a single NUMA node. The reserved CPUs can also be distributed among all NUMA nodes.
+
+[NOTE]
+====
+The reserved and isolated CPU pools must not overlap and together must span all available cores in the worker node.
+====
+
+To ensure that housekeeping tasks and workloads run on separate NUMA nodes, specify two groups of CPUs in the `spec` section of the performance profile. 
 
 * `isolated` - The CPUs for the application container workloads. These CPUs have the lowest latency. Processes in this group have no interruptions and can, for example, reach much higher DPDK zero packet loss bandwidth.
 
-* `reserved` - The CPUs for the cluster and operating system housekeeping duties, including pod infra containers. Threads in the `reserved` group tend to be very busy, so latency-sensitive applications should be run in the `isolated` group. See link:https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Create a pod that gets assigned a QoS class of `Guaranteed`].
+* `reserved` - The CPUs for the cluster and operating system housekeeping duties, including pod infra containers. Threads in the `reserved` group tend to be under higher load, so latency-sensitive applications should be run in the `isolated` group. See link:https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Create a pod that gets assigned a QoS class of `Guaranteed`].
```

Comment:
The change here is not entirely accurate. The threads are not necessarily overloaded, but there is a lot of "churn" on them. Lots of random processes.

Reply:
Updated as requested.